src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -165,6 +165,11 @@ module.exports = function<T, P, I, TI, C>(config : HostConfig<T, P, I, TI, C>) {
       const current = finishedWork.alternate;
       commitWork(current, finishedWork);
     }
+    // if the root is a HostContainer, it may have a callback.
+    if (finishedWork.tag === HostContainer) {
```

Comment:
This tag will be checked inside of commitLifeCycles so there is no need to do it here. You can just remove the conditional.